### PR TITLE
Add link to Unified Plan Transition Guide

### LINKED ISSUE #207: Add link to Unified Plan migration guide
Add link to Unified Plan migration guide

### PR DESCRIPTION
**Pull request by henbos**
Fixes #207

### CODE REVIEW

File: web-apis/chrome/unified-plan/index.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ We will be monitoring the usage of the flag, and the amount of SDP being receive
...
 In this phase, the sdpSemantics flag and all code for supporting Plan B will be removed from Chrome. Setting the sdpSemantics flag will not be an error, but will have no effect.
 
-## Further information
+## Preparing Your Application For Unified Plan
 
-For information on Unified Plan in the C++ API, see the C++ API transition plan (forthcoming).
+For detailed information about Plan B and Unified Plan differences and how your application may need to be updated in preparation for Unified plan, see the [“Unified Plan” Transition Guide (JavaScript)][1].
 
-For detailed information on how to modify your application for Unified plan, see the Web API transition guide (forthcoming).
+[1] https://docs.google.com/document/d/1-ZfikoUtoJa9k-GZG1daN0BU3IjIanQ_JSscHxQesvU/edit?usp=sharing
```
Comment (alvestrand) [inline web-apis/chrome/unified-plan/index.md:63]: Does this URL style produce correct results (clickable links) in Jekyll?
I thought you had to do [reftext](url)

Comment (henbos) [reply web-apis/chrome/unified-plan/index.md:63]: I looked at how links were handled in web-apis/chrome/index.md and tried to do the same, so I think the answer is yes, but I haven't figured out how to test this locally.

**Review by alvestrand — APPROVED** [review]

**Comment by henbos** [comment]
I was wrong, the link did not work! I'll look into it tomorrow.

**Comment by henbos** [comment]
Still better than nothing :)

**Pull request merged**